vendor/k8s.io/kubernetes/staging/src/k8s.io/apimachinery/pkg/runtime/serializer/json/json.go

```diff
@@ -187,7 +189,8 @@ func (s *Serializer) Decode(originalData []byte, gvk *schema.GroupVersionKind, i
 		return nil, actual, err
 	}
 
-	if err := jsoniter.ConfigFastest.Unmarshal(data, obj); err != nil {
+	// if err := jsoniter.ConfigFastest.Unmarshal(data, obj); err != nil {
+	if err := json.Unmarshal(data, obj); err != nil {
```

Comment:
this should use utiljson

Reply:
fixed here https://github.com/soltysh/origin/pull/17